### PR TITLE
feat(distribution): add functionality to reopen an inquiry

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: packages/distribution/addon/components/cd-inquiry-dialog.js [hunk 1]
```diff
@@ -68,11 +70,42 @@ export default class CdInquiryDialogComponent extends Component {
...
+       * now not anymore. This indicates that the work item was redone which
+       * should result in a transition to the answer view.
+       */
+      const redoableChangedToTrue = this._redoableStates.find((stateObj) => {
+        const updatedWorkItem = allWorkItems.edges.find(
+          (edge) => decodeId(edge.node.id) === stateObj.id
+        );
+
+        return (
+          stateObj.isRedoable && !(updatedWorkItem?.node.isRedoable ?? true)
```
Comment (luytena) [inline packages/distribution/addon/components/cd-inquiry-dialog.js:85]: Which case is handled by the `?? true`? Is the `isRedoable` property reset in certain work-items?

Comment (anehx) [reply packages/distribution/addon/components/cd-inquiry-dialog.js:85]: If `updatedWorkItem` is `undefined`

Comment (luytena) [reply packages/distribution/addon/components/cd-inquiry-dialog.js:85]: Ah yes, makes sense, thanks.

Comment (luytena) [reply packages/distribution/addon/components/cd-inquiry-dialog.js:85]: Minor nitpick: Shouldn't the name be `redoableChangedToFalse` or `redoneWorkItem`, if we're looking for work-items that are no longer redoable and have been redone?

**Comment by luytena** [comment]
Otherwise looks good to me :ok_hand:

File: packages/distribution/addon/components/cd-inquiry-dialog.js (hunk 1 above)
Comment (anehx) [reply packages/distribution/addon/components/cd-inquiry-dialog.js:85]: Valid point, changed it to `redoneWorkItem`

**Review by luytena — APPROVED** [review]

**Pull request merged**